content/community/events/osdc-2019-05.md

```diff
@@ -0,0 +1,12 @@
+---
+event: Open Source Data Center Conference
+topic: KUDO - Kubernetes Operators, the easy way
+speaker: Matt Jarvis
+date: 2019-05-14
+location: Berlin, DE
+url: https://youtu.be/qAUmRfbd300
```

Comment:
@mattj-io great idea to link a recording – I could easily add a `recording` property which would allow us to render something like
> (event details) (recording)

with the appropriate links. WDYT?

Reply:
Sounds good. On the DC/OS website we always linked to the event until the recording was available and then linked to the recording, but having another field would also work. We could also consider making the material available via slideshare or something - although for Kudo most of them are probably identical since everyone is using my deck ;)